src/Homework1.java

```diff
@@ -1,101 +1,170 @@
-                System.out.println("Wrong number: " + weekDay);
+        System.out.println("Sum for range "+ a +" to " +b +":"+sum2);
+        System.out.println("Average for range"+ a +" to " +b +":" + sum2/c);
+
+     //problem 3
+        System.out.println("3.Please insert a target number for Fibonacci series ");
+        int x3=scan.nextInt();
+        if(x3==1)
+            System.out.println("The first Fibonacci number is 1 and the average is 1");
+        else if(x3==2)
+            System.out.println("The first 2 Fibonacci numbers are 1 1 and the average is 1");
+        else
+        {
+            int sum3=2,c3=2;
+            int f1,f2,f3;
+            f1=1;
+            f2=1;
+            System.out.print("The first " + x3 + " Fibonacci numbers are :" + f1 + " " + f2 );
+            do{
+                f3=f1+f2;
+                f1=f2;
+                f2=f3;
+                sum3=sum3+f3;
+                c3++;
+                System.out.print(" " + f3);
+                x3--;
+            }while(x3>2);
+            float avg3= (float) sum3/c3;
+            System.out.println("\nThe average is " + avg3);
         }
 
-
-        // Loop blocks
-
-        int[] ints = {1, 2, 3, 4, 5};
-        // Calculate sum of all numbers
-        int sum = ints[0] + ints[1] + ints[2] + ints[3] + ints[4];
-
-        //for
-        sum = 0;
-        for (int i = 0; i < 5; i++) {
-            //code block
-            sum = sum + ints[i];
+     //problem 4
+        System.out.println("4.Please insert array size");
+        int n4=scan.nextInt();
+        System.out.println("Insert numbers");
+        int[] v =new int[n4];
+        for(int i=0;i<n4;i++)
+            v[i]=scan.nextInt();
+        if(v[0]==v[n4-1]) System.out.println("TRUE");
+        else System.out.println("FALSE");
+
+     //problem 5
+        System.out.println("5.Please insert first array size");
+        int n5_1=scan.nextInt();
+        System.out.println("Insert first array");
+        int[] v5_1 =new int[n5_1];
+        for(int i=0;i<n5_1;i++)
+            v5_1[i]=scan.nextInt();
+        System.out.println("Please insert second array size");
+        int n5_2=scan.nextInt();
+        System.out.println("Insert second array");
+        int[] v5_2 =new int[n5_2];
+        for(int i=0;i<n5_2;i++)
+            v5_2[i]=scan.nextInt();
+        if(n5_1!=n5_2) System.out.println("FALSE");
+        else
+        {
+            int ok=1;
+            for(int i=0;i<n5_1;i++)
+                if(v5_1[i]!=v5_2[i]) ok=0;
+            if(ok==1) System.out.println("TRUE");
+            else  System.out.println("FALSE");
         }
-        System.out.println("Veniturile mele sunt: " + sum);
 
-        int j = 0;
-        sum = 0;
-        while (j < ints.length) {
-            sum = sum + ints[j];
-            j = j + 1;
-//            j++;
-//            j+=1;
+    //problem 6
+        System.out.println("6.Please insert first array size");
+        int n6_1=scan.nextInt();
```

Comment:
Incearca pe viitor sa denumesti variabilele mai sugestiv. Incurajam sa folositi cuvinte in numele variabilelor pentru a defini exact ce fac

Reply:
O sa tin cont. Multumesc!